python/tvm/contrib/hexagon/build.py

```diff
@@ -467,6 +467,10 @@ def _terminate_remote(self):
             self._adb_device_sub_cmd + ["shell", f"kill `cat {self._workspace}/rpc_pid.txt`"]
         )
 
+    def _cleanup_directory(self):
+        # Cleanup directory
```

Comment:
change to """Remove workspace directory on remote target"""

Reply:
Done. Thanks